src/oas2/transformers/getExamplesFromSchema.ts

```diff
@@ -0,0 +1,16 @@
+import { isObject } from 'lodash';
+import { Schema } from 'swagger-schema-official';
+
+export function getExamplesFromSchema(data: Schema) {
+  return isObject(data)
```

Comment:
This is always true by definition (`data: Schema`) — this check is no longer necessary